app/__tests__/Button.test.js

```diff
@@ -0,0 +1,19 @@
+import { render } from "@testing-library/react";
+import Button from "@/components/common/buttons/Button";
+
+describe("Button Component", () => {
+  test("renders a button with href prop and navigates to the correct URL", () => {
+    render(<Button href="/test-url" label="Test Link" />);
```

Comment:
href should be a variable and used in the test as well

Reply:
thank you for the suggestion, done!